manim/mobject/coordinate_systems.py

```diff
@@ -495,3 +503,279 @@ def get_coordinate_labels(self, *numbers, **kwargs):
+        Specifies a default labelling system for the azimuth. Choices are:
+
+            - ``"PI radians"``: Fractional labels in the interval :math:`\left[0, 2\pi\right]` with :math:`\pi` as a constant.
+            - ``"TAU radians"``: Fractional labels in the interval :math:`\left[0, \tau\right]` (where :math:`\tau = 2\pi`) with :math:`\tau` as a constant.
+            - ``"degrees"``: Decimal labels in the interval :math:`\left[0, 360\right]` with a degree (:math:`^{\circ}`) symbol.
+            - ``"gradians"``: Decimal lables in the interval :math:`\left[0, 400\right]` with a superscript "g" (:math:`^{g}`).
+
+        .. manim:: PolarPlaneUnits
+            :ref_classes: PolarPlane
+
+            polarplot_pi = PolarPlane(azimuth_units="PI radians").scale(0.4).add_coordinates()
+            polarplot_tau = PolarPlane(azimuth_units="TAU radians").scale(0.4).add_coordinates()
+            polarplot_degrees = PolarPlane(azimuth_units="degrees").scale(0.4).add_coordinates()
+            polarplot_gradians = PolarPlane(azimuth_units="gradians").scale(0.4).add_coordinates()
+
+            class PolarPlaneUnits(Scene):
+                def construct(self):
+                    self.add(polarplot_pi)
+                    self.wait(2)
+                    self.play(FadeOutAndShift(polarplot_pi, LEFT))
+                    self.play(FadeInFrom(polarplot_tau, RIGHT))
+                    self.wait(2)
+                    self.play(FadeOutAndShift(polarplot_tau, LEFT))
+                    self.play(FadeInFrom(polarplot_degrees, RIGHT))
+                    self.wait(2)
+                    self.play(FadeOutAndShift(polarplot_degrees, LEFT))
+                    self.play(FadeInFrom(polarplot_gradians, RIGHT))
+                    self.wait(2)
+
+
+    azimuth_offset : :class:`float`, optional
+        The angle offset of the azimuth labels, expressed in radians.
+
+    azimuth_direction : :class:`str`, optional
+        The direction of the azimuth labels.
+
+            - ``"CW"``: Clockwise.
+            - ``"CCW"``: Anti-clockwise.
+
+    azimuth_buff : :class:`int`, optional
+        The buffer for the azimuth labels.
+
+    radius_config: :class:`dict`, optional
+        The axis config for the radius.
+
+    radius_max : :class:`float`, optional
+        The maximum value of the radius.
+    """
+
+    def __init__(
+        self,
+        color=BLUE,
+        azimuth_line_frequency=None,
+        radius_line_frequency=1,
+        azimuth_units="PI radians",
+        azimuth_offset=0,
+        azimuth_direction="CCW",
+        azimuth_buff=SMALL_BUFF,
+        radius_config={"label_direction": DOWN + LEFT},
+        radius_max=None,
+        **kwargs,
+    ):
+        if azimuth_units in [
+            "PI radians",
+            "TAU radians",
+            "degrees",
+            "gradians",
+            None,
+        ]:
+            self.azimuth_units = azimuth_units
+        else:
+            ValueError(
```

Comment:
If you want to throw an error, you have to `raise` it.

```suggestion
            raise ValueError(
```